### PR TITLE
horizontal emoticon style interface option

### PR DESCRIPTION
**Pull request by Tsunder**
rescales morale changes to be more responsive and useful at a glance, -50 is a threshold for crafting
mod also adds emoticons for avian mutation threshold

### CODE REVIEW

**Comment by AMurkin** [comment]
Why don't you put values into mod's json file? Thus, providing the possibility of creating more than one such mod.

**Comment by kevingranade** [comment]
"requested non-existing option THREE_CHARACTER_MORALE
File src/options.cpp
Line 2046

You need to add an options entry.
Also, you need to test your changes -n-

**Comment by Tsunder** [comment]
added

@n@ compiling is hard

File: src/options.cpp [hunk 1]
```diff
@@ -1071,6 +1071,12 @@ void options_manager::init()
         { { "numbers", translate_marker( "Numbers" ) }, { "bars", translate_marker( "Bars" ) } }, "bars"
         );
 
+    add( "MORALE_STYLE", "interface", translate_marker( "Morale style" ),
+        translate_marker( "Morale display style in sidebar." ),
+        //~ aim bar style - bars or numbers
```
Comment (Night-Pryanik) [inline src/options.cpp]: Uhmm, this note to translators is obviously wrong.

**Comment by Leland** [comment]
Linking #22318 for posterity

**Pull request merged**